src/bp-templates/bp-nouveau/js/buddypress-media.js

```diff
@@ -6204,7 +6204,6 @@ window.bp = window.bp || {};
 			var currentVideo =  document.getElementById( $( '.bb-media-model-wrapper.video video' ).attr('id') );
 			if( currentVideo ) {
 				currentVideo.pause();
-				currentVideo.src = '';
```

Comment:
Same functionalitu we have when user open document theater. Because there is a case when user has a document in comment area and opens it then also needs to stop playing video.

Reply:
This change does not require because closeTheater function changes fix the issue already.